build.gradle

```diff
@@ -1622,7 +1622,6 @@ project(':streams:test-utils') {
 
   javadoc {
     include "**/org/apache/kafka/streams/test/**"
-    exclude "**/org/apache/kafka/streams/internals/**", "**/org/apache/kafka/streams/**/internals/**"
```

Comment:
As the related class have be moved from `internals`, does it need to remove this `exclude` rule?

Reply:
The files I converted to static inner classes were the only contents of these packages, so they don't exist anymore. I figured I'd go ahead and remove the exclusion, too.